lib/ddr/models/solr_document.rb

```diff
@@ -11,19 +11,20 @@ module SolrDocument
       alias_method :pid, :id
     end
 
-    class NotFound < Error; end
+    class NotFound < Error;
+    end
```

Comment:
Can we take out the stylistic changes in SolrDocument?  I'd like to have a separate conversation about those. The one above is just one example.